controls_doctor/_doctor.py

```diff
@@ -63,8 +63,8 @@ def nfs_perf():
     THRESH = 1000000
     output_path = os.path.expanduser('~/.check-doctor-dd-testfile')
     output = subprocess.check_output(
-            ['dd', 'if=/dev/random', 'of={}'.format(output_path),
-             'bs=1048576', 'count=1'], stderr=subprocess.STDOUT).decode()
+            ['sync;', 'dd', 'if=/dev/random', 'of={}'.format(output_path),
+             'bs=1048576', 'count=1', '; sync'], stderr=subprocess.STDOUT).decode()
```

Comment:
Following up from an in-person conversation awhile ago, I don't think the time in the second `sync` will be counted in the performance test. May something blunter like a plain old `time` command would be more robust. Accuracy is more important than precision here.